spec/concerns/asset_push_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-class AssetPushController < ApplicationController
+class AssetPushController < ActionController::Base
```

Comment:
Don't inherit from the dummy application controller – it sets up state that we don't want for our controllers, so we need to reserve it for request tests.